.github/workflows/ci.yml

```diff
@@ -28,4 +28,4 @@ jobs:
         run: npm run build
 
       - name: Run tests
-        run: npm test -- --coverage --watchAll=false
+        run: npm run test
```

Comment:
why without coverage check? is it just an option that you don't want to use anymore?
or do you need to add coverage check in package.json at test command?